src/nuclei/client/main.py

```diff
@@ -138,17 +184,22 @@ def _get_app_specification(self, app: str) -> dict:
         ConnectionError:
             Application not available
         TypeError:
-            Wrong type for `app` argument
+            Wrong type for `app` or `version` argument
         ValueError:
-            Wrong value for `app` argument
+            Wrong value for `app` or `version` argument
```

Comment:
We are missing the code to actually check this.

Reply:
This is a propagating error, its in `self.get_url`.